ILL/IDF/in16bd_generateIDF.py

```diff
@@ -1,6 +1,12 @@
-from helper import MantidGeom
-from math import cos, sin, pi
+from math import pi
 from time import strftime, gmtime
+import os
+import sys
+
+path = os.path.abspath("")
```

Comment:
I guess this assumes running from the repository root?

Could we do `sys.path.insert(0, os.path.normpath(os.path.dirname(__file__), "../../")` as we know where the file is?

Reply:
Thanks, that's a way better way of doing that.